spacy/pipeline/entity_linker.py

```diff
@@ -55,12 +66,10 @@
         "incl_context": True,
         "entity_vector_length": 64,
         "get_candidates": {"@misc": "spacy.CandidateGenerator.v1"},
```

Comment:
This should then be bumped to v2.

Reply:
See [this comment](https://github.com/explosion/spaCy/pull/12367#discussion_r1481150728).